pkg/cmd/secrets.go

```diff
@@ -147,13 +147,21 @@ func setSecrets(cmd *cobra.Command, args []string) {
 
 	secrets := map[string]interface{}{}
 	var keys []string
-	for _, arg := range args {
-		secretArr := strings.Split(arg, "=")
-		keys = append(keys, secretArr[0])
-		if len(secretArr) < 2 {
-			secrets[secretArr[0]] = ""
-		} else {
-			secrets[secretArr[0]] = secretArr[1]
+
```

Comment:
Can we add comments that one is
`secrets set FOO="bar"`
and the other one
`secrets set FOO "bar"`